Gemfile

```diff
@@ -11,4 +11,3 @@ gemspec
 # your gem to rubygems.org.
 
 # To use debugger
-# gem 'debugger'
```

Comment:
I guess you probably want to remove the line above it too?
Is there a particular reason to delete this line btw? Doesn't work or not needed in ruby 2.6.1?